### ISSUE TITLE
Structure for image files and other documents or binaries

### ISSUE THREAD

**Issue by DanielMaintrac** [on-api/on-api#40]
With the suggestions of flowcharts a need of a folder structure is needed to host these files in the repository. 
Should we just add folders like "2.4/images/" or should we prepare for a more advanced structure of future files
2.4/files/binaries/, 2.4/files/images/, 2.4/documents/

**Comment by teemufrosen** [comment]
I agree with the suggestion. The second option would be best in my opinion.

**Comment by DanielMaintrac** [comment]
Vi kanske behöver något namnschema för bildfiler också för att förhindra krockar.

**Comment by joeseos** [comment]
We used on-api/2.3.1/files/ for 2.3.1 and maybe we should stick with it.

**Comment by teemufrosen** [comment]
Working group meeting: If there is a need for subdirectories in repos, they will be added. Closing.
